main/com.spotify/dataset.tt

```diff
@@ -20,5 +20,19 @@ dataset @com.spotify language "en" {
     #[id=27924898]
     ;
 
+    action (p_playable : Entity(org.thingpedia.media-source:playable)) = @com.spotify.add_item_to_library(playable=p_playable)
+    #_[utterances=["add ${p_playable:no-undefined} to my favorite",
```

Comment:
Some of these are fine without `:no-undefined`. `no-undefined` prevents generating "some X" (in this case "some music") as replacement for the placeholder. I think "add some music to my favorite" is a useful sentence to have - the agent will ask a follow-up question in that case.